applications/vessel-history/src/features/map/info/Info.module.css

```diff
@@ -0,0 +1,118 @@
+.infoContainer {
+  width: 80%;
+  position: fixed;
+  bottom: 0;
+  padding: 19px 15px 0;
+  border-radius: 10px 10px 0 0;
+  background: linear-gradient(180deg, #e5f0f2 0%, #fff 100%);
+  box-shadow: 0 1px 2px 0 rgba(22, 63, 137, 0.2), 0 2px 10px 0 rgba(1, 37, 91, 0.15);
+  left: 50%;
+  min-height: 50px;
+  margin-left: -40%;
+  z-index: 200;
+}
+
+.handler {
+  cursor: ns-resize;
+  z-index: 10;
+  overflow: hidden;
+  display: -webkit-box;
+  display: flex;
+  -webkit-box-pack: center;
+  justify-content: center;
+  box-sizing: border-box;
+  height: 1px;
+  width: 10%;
+  border: 1px solid rgba(34, 68, 126, 0.65);
+  opacity: 0.4;
+}
+
+.description {
+  margin-bottom: 20px;
+}
+
+.panel {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.footer {
+  display: flex;
+  flex-flow: column nowrap;
+  align-items: stretch;
+  justify-content: flex-start;
+  height: auto;
+  overflow: hidden;
+  font-size: 16px !important;
+  padding-top: 13px;
+}
+
+.footer .label {
+  font-size: 16px !important;
+}
+
+.footerArea {
+  flex-grow: 1;
+  display: flex;
+  min-height: 50px;
+  overflow: scroll;
+  flex-flow: column;
+  overflow-y: auto;
+}
+
+.footerArea .ActivityModalDetails {
+  display: block;
+}
+
+.footerAreaContent {
+  max-height: 400px;
+  min-height: 50px;
+}
+
+.footerAreaContent .eventData .date {
+  font-size: 1rem;
+  line-height: var(--space-S);
+}
+
+.footerAreaContent .eventData .description {
+  font: var(--font-XS);
+  line-height: var(--space-S);
+  margin-top: var(--space-XS);
+}
+
+.dateFormat {
+  opacity: 0.5;
+  color: #22447e;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
```

Comment:
Use colors, fonts, spaces, sizes, etc  from variables that are defined in `base.css`